Specs/Core/loadKTX2Spec.js

```diff
@@ -0,0 +1,322 @@
+import { loadKTX2 } from "../../Source/Cesium.js";
+import { KTX2Transcoder } from "../../Source/Cesium.js";
+import { PixelFormat } from "../../Source/Cesium.js";
+import { Resource } from "../../Source/Cesium.js";
+import { RuntimeError } from "../../Source/Cesium.js";
+
+describe("Core/loadKTX2", function () {
+  it("throws with no url", function () {
+    expect(function () {
+      loadKTX2();
+    }).toThrowDeveloperError();
+  });
+
+  it("throws with unknown supported formats", function () {
+    var promise = KTX2Transcoder.transcode(new Uint8Array());
+    expect(promise).toBeDefined();
+    return promise.then(
+      function () {
+        return;
+      },
+      function (error) {
+        expect(function () {
+          throw error;
+        }).toThrowDeveloperError();
+      }
+    );
+  });
```

Comment:
It would be better if `KTX2Transcoder.transcode` checked that the parameters are defined instead of checking them in the worker. Generally it's good to fail fast like that. It should also simplify the test.

Also the preferred way to handle promises is to chain `.then` and `.otherwise` like below. Also `return` should be `fail()` so that jasmine fails if the promise resolves. Make sure to update other tests in this file.

```
    return promise
      .then(function (value) {
        return; // <------- this should actually be fail() so that jasmine fails if the promise somehow resolves
      })
      .otherwise(function (error) {
        expect(function () {
          throw error;
        }).toThrowDeveloperError();
      });
```